common.c

```diff
@@ -314,19 +319,13 @@ get_config_file()
 	rtr_getenv_t real_getenv;
 	int olderrno;
 
-	if (!get_tracing_enabled())
```

Comment:
Any reason for removing this? Later in this function we call strcpy and strcat and those are going to show up in the logs causing noise OR loops. Even if those don't show in the log (because they are built in in some platforms)  we are calling stuff like real_fopen which internally could potentially call anything in some platform.